release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -3453,6 +3454,27 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
 release, see xref:../upgrading/index.adoc#install-config-upgrading-index[Upgrade
 methods and strategies] for instructions.
 
+The 3.11.248 release includes two separate package advisories which were
+released at different times. The
+link:https://access.redhat.com/errata/RHBA-2020:3416[RHBA-2020:3416] advisory
+includes a bug fix for CRI-O, which is summarized in
+link:https://bugzilla.redhat.com/show_bug.cgi?id=1866132[BZ#1866132]. To upgrade
+your cluster with the latest CRI-O package update, complete the following steps:
+
+. Clean all nodes:
++
+[source,terminal]
+----
+# yum clean all
+----
+
+. Run the xref:../upgrading/automated_upgrades.adoc#upgrading-ocp[3.11 upgrade playbook].
+
+. Verify all nodes get the `cri-o-1.11.16-0.11.dev.rhaos3.11.gitd6a416d.el7`
```

Comment:
@codyhoag Not sure how to reword, but _get_ seems odd here.
_Verify that the  `cri-o-1.11.16-0.11.dev.rhaos3.11.gitd6a416d.el7` package was downloaded to each node_?

Do I need to verify the nodes got the second package?

Reply:
yum list cri-o
verify version is 1.11.16-0.11.dev.rhaos3.11.gitd6a416d.el7

Reply:
Just updated, Thanks!